File: app/src/main/java/com/BlizzardArmory/warcraft/Media/Media.java

package com.BlizzardArmory.warcraft.Media;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Media {

    @SerializedName("_links")
    @Expose
    private Links links;
    @SerializedName("assets")
    @Expose
    private List<Asset> assets = null;

    public Links getLinks() {
        return links;
    }

    public void setLinks(Links links) {
        this.links = links;
    }

    public List<Asset> getAssets() {
        return assets;
    }

    public void setAssets(List<Asset> assets) {
        this.assets = assets;
    }

}

File: app/src/main/java/com/BlizzardArmory/warcraft/media/Media.kt
package com.BlizzardArmory.warcraft.media

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Media {
    @SerializedName("_links")
    @Expose
    var links: Links? = null

    @SerializedName("character")
    @Expose
    var character: Character? = null

    @SerializedName("avatar_url")
    @Expose
    var avatarUrl: String? = null

    @SerializedName("bust_url")
    @Expose
    var bustUrl: String? = null

    @SerializedName("render_url")
    @Expose
    var renderUrl: String? = null

}